### PR TITLE
180915373 filters aggregations notes

### PR DESCRIPTION
**Pull request by Adorism**
## Change description

Documentation frequently linked to the filters section of core product concepts which needed some more information to be a helpful link. Also the sources refer to aggregation methods and core product concepts fits as a place to link for more information better than the section previously linked to for aggregation info. 

### Development

- [X] Application changes have been tested appropriately

### Impact

- [ ] Code follows company security practices and guidelines
- [ ] Security impact of change has been considered
- [ ] Performance impact of change has been considered
- [ ] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [ ] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: pages/docs/getting-started/product-concepts.mdx [hunk 1]
```diff
@@ -76,7 +94,19 @@ See [our Integrations page](/integrations) for an up-to-date list of our support
...
 ## Filter
 
-Filters are a series of rules used to filter data into some desired subset.
+Filters are a series of rules used to filter data into some desired subset. Sometimes you may want to specifically include or exclued rows that meet a criteria. This is where Filters come in.
+
+Each filter rule has a key and operation. The key is a property name, and the operation is a comparison operator.
+
+The following are operations available in Grouparoo's filtering options:
+
+- `equals`
```
Comment (teallarson) [inline pages/docs/getting-started/product-concepts.mdx]: For these, maybe we want to put the version that would be in the config files as well.  They use a more abbreviated version.  So:
- `eq`: equals
- `ne`: does not equal
- etc.

They're listed on this page https://www.grouparoo.com/docs/support/config-files what do you think?

**Review by edmundito — COMMENTED** [review]
Looks good

File: pages/docs/getting-started/product-concepts.mdx (hunk 1 above)
Comment (Adorism) [reply pages/docs/getting-started/product-concepts.mdx]: @tealjulia are substring, notSubstring, and in still used by any plugins? I have checked a few sources.

Comment (teallarson) [reply pages/docs/getting-started/product-concepts.mdx]: Yep.  BigQuery, Clickhouse, Mongo, MySQL, Postgres, Snowflake, and SQLite.

**Review by teallarson — APPROVED** [review]

**Pull request merged**